src/libs/Notification/PushNotification/NotificationType.ts

```diff
@@ -1,30 +1,38 @@
+import type {ValueOf} from 'type-fest';
 import type {OnyxServerUpdate} from '@src/types/onyx/OnyxUpdatesFromServer';
 
 const NotificationType = {
     REPORT_COMMENT: 'reportComment',
+    MONEY_REQUEST: 'moneyRequest',
 } as const;
 
 type NotificationDataMap = {
-    [NotificationType.REPORT_COMMENT]: ReportCommentNotificationData;
+    [NotificationType.REPORT_COMMENT]: ReportCommentPushNotificationData;
+    [NotificationType.MONEY_REQUEST]: ReportActionPushNotificationData;
 };
 
-type NotificationData = ReportCommentNotificationData;
+type PushNotificationData = ReportCommentPushNotificationData | ReportActionPushNotificationData;
 
-type ReportCommentNotificationData = {
+type BasePushNotificationData = {
     title: string;
-    type: typeof NotificationType.REPORT_COMMENT;
-    reportID: number;
-    reportActionID: string;
-    shouldScrollToLastUnread?: boolean;
-    roomName?: string;
+    type: ValueOf<typeof NotificationType>;
     onyxData?: OnyxServerUpdate[];
     lastUpdateID?: number;
     previousUpdateID?: number;
 };
 
+type ReportActionPushNotificationData = BasePushNotificationData & {
+    reportID: number;
+    reportActionID: string;
+};
+
+type ReportCommentPushNotificationData = ReportActionPushNotificationData & {
+    roomName?: string;
+};
+
```

Comment:
I worry that this won't be obvious to anyone modifying the backend. I'll save some comments for the Web PR.